core/src/main/java/com/linecorp/armeria/common/HttpStatus.java

```diff
@@ -491,6 +491,31 @@ public boolean isContentAlwaysEmpty() {
         return isContentAlwaysEmpty(code);
     }
 
+    /** Returns whether the {@link HttpStatus} is an information, with a status code of 1XX. */
+    public boolean isInformation() {
+        return codeClass().equals(HttpStatusClass.INFORMATIONAL);
+    }
+
+    /** Returns whether the {@link HttpStatus} is a success, with a status code of 2XX. */
+    public boolean isSuccess() {
+        return codeClass().equals(HttpStatusClass.SUCCESS);
+    }
+
+    /** Returns whether the {@link HttpStatus} is a redirection, with a status code of 3XX. */
+    public boolean isRedirection() {
+        return codeClass().equals(HttpStatusClass.REDIRECTION);
+    }
+
+    /** Returns whether the {@link HttpStatus} is a client error, with a status code of 4XX. */
+    public boolean isClientError() {
```

Comment:
As I mentioned in the issue I don't find a lot of use in specifying client vs server, or convenience for information, which is so rare. But this is non-blocking and happy with what others think

Reply:
I think it's OK to have all of `is{Error,ServerError,ClientError}`.

Reply:
@mercanil I think you omitted the `isError` method. It could be
```java
boolean isError() {
	return isClientError() || isServerError();
}
```

Reply:
Thank you for suggestion @ikhoon but being more specific would be better. I prefer to keep as is.

Reply:
I'd prefer to have `isError()` for convenience. :pray:

Reply:
added isError() method. Thanks for advice.